ocpairprogramming/src/main/java/com/pino/project/ocpairprogramming/java8/ocp/chapter7/concurrency/concurrentcolletions/ObtainSynchronizedCollection.java

```diff
@@ -0,0 +1,55 @@
+package com.pino.project.ocpairprogramming.java8.ocp.chapter7.concurrency.concurrentcolletions;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+* ObtainSynchronizedCollection
+* 
+* Table 7.12 Synchronized collections methods
+* 
+*   Method Name
+_________________________________________________
+	synchronizedCollection(Collection<T> c)
+	synchronizedList(List<T> list)
+	synchronizedMap(Map<K,V> m)
+	synchronizedNavigableMap(NavigableMap<K,V> m)
+	synchronizedNavigableSet(NavigableSet<T> s)
+	synchronizedSet(Set<T> s)
+	synchronizedSortedMap(SortedMap<K,V> m)
+	synchronizedSortedSet(SortedSet<T> s)
+                                                         
+--------------------------------------------------
+*  :-) Guarantee mutual access
+ *
+ * :-( It doesn't work with iterator, which must be enclosed in a synchronized block. 
+ *     Unlike concurrent collection, the synch. collection throws an exception, if they it is modified within an iterator by a single thread
```

Comment:
personally I didn't understand why it throws ConcurrentModificationException, if the new collection is intended to be synchronized.